tests/test_service_case_helpers.py

```diff
@@ -29,3 +30,27 @@ def test_add_route(self):
                     json.loads(response.body.decode("utf-8")))
 
         self.execute_case(BasicTest)
+
+    def test_service_case_helpers_add_service_allows_optional_argument(self):
```

Comment:
The test name should say what the optional argument is. maybe `accepts_existing_service` instead of `allows_optional_argument`?